bvm/ballerina-runtime/src/main/resources/MessagesBundle.properties

```diff
@@ -165,5 +165,6 @@ illegal.array.insertion = array of length {0} cannot be expanded into array of l
 illegal.tuple.insertion = tuple of length {0} cannot be expanded into tuple of length {1} without filler values
 unexpected.xml.type = lang.xml function ''{0}'' only accept xml type ''{1}''
 java.null.reference = handle refers to Java null
-illegal.array.size = cannot change length of fixed length array to ''{0}''
-cannot.change.tuple.size = cannot change tuple size
+illegal.array.size = cannot change the length of an array of fixed length ''{0}'' to ''{1}''
+illegal.tuple.size = cannot change the length of a tuple of fixed length ''{0}'' to ''{1}''
+illegal.rest.tuple.size = cannot change the length of tuple with ''{0}'' fixed member types to ''{1}''
```

Comment:
```suggestion
illegal.rest.tuple.size = cannot change the length of a tuple with ''{0}'' fixed member(s) to ''{1}''
```

Reply:
I think we need to keep the 'type' as these members are not fixed rather mandatory